appserver/admingui/common/src/main/java/org/glassfish/admingui/common/handlers/NewSSLHandlers.java

```diff
@@ -126,100 +129,85 @@ public static void convertCiphersItemsToStr(HandlerContext handlerCtx) {
     }
 
 
-        
-        private static String[] getSelectedCiphersList(String selectedCiphers){
-            Vector selItems = new Vector();
-            if(selectedCiphers != null){
-                String[] sel = selectedCiphers.split(","); //NOI18N
-                for(int i=0; i<sel.length; i++){
-                    String cName = sel[i];
-                    if(cName.startsWith("+")){ //NOI18N
-                        cName = cName.substring(1, cName.length());
-                        selItems.add(cName);
-                    }
-                }    
-            }
-            return (String[])selItems.toArray(new String[selItems.size()]);
-        }
-        
-        private static String processSelectedCiphers(String[] selectedCiphers, String ciphers){
-            StringBuilder sb = new StringBuilder();
-            String sep = "";
-            if ( ! GuiUtil.isEmpty(ciphers)){
-                sb.append(ciphers);
-                sep = ",";
-            }
-            if(selectedCiphers != null){
-                for (int i = 0; i < selectedCiphers.length; i++) {
-                    sb.append(sep).append("+").append(selectedCiphers[i]);
-                    sep = ",";
-                }
-            }
-            return sb.toString();
-        }
 
-        private static Vector getCiphersVector(String[] allCiphers){
-            Vector ciphers = new Vector();
-            if (allCiphers != null){
-                for(int i=0; i<allCiphers.length; i++){
-                    ciphers.add(allCiphers[i]);
+    private static String[] getSelectedCiphersList(String selectedCiphers){
+        List<String> selItems = new ArrayList<>();
+        if(selectedCiphers != null){
+            String[] sel = selectedCiphers.split(","); //NOI18N
```

Comment:
We try to avoid short names or abbreviations for variable names. Maybe this can just be inlined in the `for` loop?